src/routing/Chapter.tsx

```diff
@@ -8,14 +8,20 @@ import { ChapterProps } from './types';
  * @beta
  */
 export default function Chapter(props: ChapterProps): JSX.Element {
+  const listOfPages = props.children.filter((child) =>
+    String(child.type).includes('Page')
+  );
+
   return (
-    <Route path={props.path}>
-      <div className="vads-u-display--flex vads-u-align-items--center vads-u-flex-direction--column">
-        <h1>{props.title}</h1>
-      </div>
-      <Switch>
-        <>{props.children}</>
-      </Switch>
-    </Route>
+    <div>
```

Comment:
Updated to router v6 conventions, but I mentioned in my overall comment we will most likely need to rewrite this.